amivapi/tests/auth/test_link_methods.py

```diff
@@ -357,23 +360,35 @@ def get_user_methods(self, response):
             if links['href'] == 'users':
                 return links['methods']
 
+    def test_allowed(self, **args):
```

Comment:
This is now also a test function, I guess it should be a utility function instead.
Also you could specify the arguments.